pkg/reader/ccip.go

```diff
@@ -775,6 +775,81 @@ func (r *ccipChainReader) GetContractAddress(contractName string, chain cciptype
 	return addressBytes, nil
 }
 
+// GetLinkPriceUSD gets the LINK price in USD from the FeeQuoter contract on the destination chain.
+func (r *ccipChainReader) GetLinkPriceUSD(ctx context.Context) (cciptypes.BigInt, error) {
+	// Exit without an error if we cannot read the destination.
```

Comment:
This comment contradicts the code below - do we wanna exit w/ and error or without?